core/src/main/scala/org/apache/spark/ui/WebUI.scala

```diff
@@ -116,15 +116,15 @@ private[spark] abstract class WebUI(
    * @param path Path in UI to unmount.
    */
   def removeStaticHandler(path: String): Unit = {
-    handlers.find(_.getContextPath() == path).foreach(detachHandler)
+    handlers.find(_.getContextPath == path).foreach(detachHandler)
```

Comment:
@dbolshak this still isn't addressing comments that have been made several times. You can appreciate why I don't think it's worth the time to make changes like this given how much discussion it takes. Here's another example. Please read _all_ previous comments and address all of them or close this.